### DISCUSSION TITLE
rdk_doc/en/Quick_start/install_os

### DISCUSSION THREAD

**Discussion by giscus** [D-Robotics/rdk_doc#24] [General]
# rdk_doc/en/Quick_start/install_os

Before using the RDK X3 development board, the following preparations need to be made.

http://en.developer.d-robotics.cc/rdk_doc/en/Quick_start/install_os

**Comment by baqwas** [comment]
Need to specify how to check for file integrity after download (for new users). Which hashing algorithm should the end-user use for validation?

**Reply by xuguosheng110** [reply]
hello , md5 check is ok 
![image](https://github.com/user-attachments/assets/62ff2cc0-fe29-4e24-ae8f-50763de1674d)

**Reply by baqwas** [reply]
OK. Thanks again. Regards.
